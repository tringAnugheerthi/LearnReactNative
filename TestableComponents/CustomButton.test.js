import React from 'react';
import {shallow} from 'enzyme';
import CustomButton from './CustomButton';

describe('CustomButton', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const component = shallow(<CustomButton label="test label"/>)
            expect(component).toMatchSnapshot()
        });
    });
});