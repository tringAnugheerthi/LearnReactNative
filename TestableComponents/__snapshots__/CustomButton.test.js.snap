// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
